snippets/saved_replies.md

```diff
@@ -59,7 +57,7 @@ As always, be careful when force-pushing.
 
 Thanks for suggesting this change. Unfortunately, we are currently not accepting new integrations.
 
-We can only provide the stability and performance we want to provide if we can test integrations in an automated and scalable fashion. For this reason, we are suggesting people integrate with the help of our generic interfaces by either integrating natively or putting a rewriting proxy in the middle. We would be more than happy to list this on our integrations page.
+We can only provide the stability and performance we want to provide if we can properly maintain the codebase. This includes, amongst others, test integrations in an automated and scalable fashion. For this reason, we are suggesting people integrate with the help of our generic interfaces. We would be more than happy to list this on our integrations page.
```

Comment:
"this" is unclear in last sentence. It also implies we'll automatically accept everything onto that list.

We have an integrations page on which integrations using our generic interfaces are listed.